File: 04-07-2025/RazorPayIntegration/DotnetAPI/Models/Payment.cs
namespace DotnetAPI.Models
{
    public class Payment
    {
        public Guid Id { get; set; }
        public decimal Amount { get; set; }
        public string Currency { get; set; } = string.Empty;
        public string Status { get; set; } = string.Empty;
        public string RazorpayPaymentId { get; set; } = string.Empty;
        public Order? Order { get; set; }
        public Guid OrderId { get; set; }
    }
}

File: 04-07-2025/RazorPayIntegration/AngularApp/src/app/features/payment/models/payment.ts
import { OrderModel } from "../../order/models/order";

export class PaymentModel {
  constructor(
    public id: string,
    public amount?: number,
    public currency?: string,
    public status?: string,
    public razorpayPaymentId?: string,
    public order?: OrderModel,
  ) {}
}